### PR TITLE
Fever integer type for last_refreshed_on_time

### LINKED ISSUE #2940: [fever-api] mixed string and number in time
now the exact opposite to #2934 :
when looking at https://feedafever.com/api the returned `last_refreshed_on_time` should always be a number. FreshRSS-Fever is always returning a `String`.

### PR DESCRIPTION
**Pull request by Alkarex**
#Fix https://github.com/FreshRSS/FreshRSS/issues/2940
https://feedafever.com/api

### CODE REVIEW

File: p/api/fever.php [hunk 1]
```diff
@@ -288,7 +288,7 @@ public function wrap($status, array $reply = array())
 		$arr = array('api_version' => self::API_LEVEL, 'auth' => $status);
 
 		if ($status === self::STATUS_OK) {
-			$arr['last_refreshed_on_time'] = (string) $this->lastRefreshedOnTime();
```
Comment (Alkarex) [inline p/api/fever.php:291]: @kevinpapst It was [a long time ago](https://github.com/FreshRSS/FreshRSS/pull/1836), but do you happen to remember why this was casted to a string?
https://github.com/FreshRSS/FreshRSS/issues/2940

Comment (kevinpapst) [reply p/api/fever.php:291]: Hahaha, can you remember why you did things last months or even two years ago? 😁 

Short answer: no, unfortunately I can't 😞 looks correct when checking the API definition.
I would question if there was a bug at any point or if that change is only happening to be compatible with the "official documentation". I never had troubles with any clients I tested.

Offtopic: I'd really like to help more, but I don't have the time to take care of any other projects that my ones currently... sorry that you have to take care of the API now! I know its a burden (but still great for the iOS users).

**Review by kevinpapst — APPROVED** [review]

File: p/api/fever.php (hunk 1 above)
Comment (Alkarex) [reply p/api/fever.php:291]: No worries, and thanks for the quick answer :-)
And if you are still using it, could you try that it still works with your favourite app after this fix? I can merge in the master branch if that can help.
Also, I am making a table overview of clients, and feedback welcome (I have nothing iOS to try myself) https://github.com/FreshRSS/FreshRSS/pull/2942

**Comment by Alkarex** [comment]
Second take https://github.com/FreshRSS/FreshRSS/pull/2945

**Pull request merged**